src/components/LegacyPortal.js

```diff
@@ -0,0 +1,59 @@
+// This file is a fallback for a consumer who is not yet on React 16
+// as createPortal was introduced in React 16
+// Extracted from https://github.com/tajo/react-portal
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+
+export default class Portal extends React.Component {
+  componentDidMount() {
+    this.renderPortal();
+  }
+
+  componentDidUpdate() {
+    this.renderPortal();
+  }
+
+  componentWillUnmount() {
+    ReactDOM.unmountComponentAtNode(this.defaultNode || this.props.node);
+    if (this.defaultNode) {
+      document.body.removeChild(this.defaultNode);
+    }
+    this.defaultNode = null;
+    this.portal = null;
+  }
+
+  renderPortal() {
+    if (!this.props.node && !this.defaultNode) {
```

Comment:
can we deconstruct props at the begging of this function? because we use `this.props` a lot in this function.
( `let { node, children } = this.props` )